pkg/plugin/util/snapshot.go

```diff
@@ -0,0 +1,56 @@
+package util
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Snapshot stores information about an item snapshot taken as
+// part of a Velero backup.
+type Snapshot struct {
```

Comment:
Why do you need to add this in utils?  This seems redundant with the Snapshot CRD in backupdriver.

Reply:
Currently, yes. However, in our initial discussion, we didn't plan to piggy-back the snapshot blob on PVC annotation in Velero backup. So, we have to somehow correlate the snapshotID and backupName. We cannot do that using backdriver Snapshot CRD. That's why this Snapshot object is defined here with BackupName available in its spec. I will try to do some test about using Snapshot CR object instead. Before the verification is done, I will leave this Snapshot object definition as it is.